src/inventory/inventory-landing-page.jsx

```diff
@@ -39,12 +44,43 @@ export const InventoryLandingPage = () => {
 		setSearchText(event.target.value);
 	};
 
+	const exportToExcel = () => {
```

Comment:
can exportToExcel function can be moved to a separate file, as its not the responsibility of inventory-landing-page to export it to a xlsx?